### PR TITLE
[Workers] Update Cache API limits

### LINKED ISSUE #11787: Cache API Calls Limit on Workers Standard Pricing Plan
### Which Cloudflare product does this pertain to?

Workers

### Existing documentation URL(s)

https://developers.cloudflare.com/workers/platform/limits/#cache-api-limits

### What changes are you suggesting?

This table shows the limits for Bundled Workers, but not for Unbound or Standard. The footnote says it uses the same quota as fetch, so I'm guessing/hoping that it is 1000 calls/request. 

### Additional information

image

### PR DESCRIPTION
**Pull request by Erisa**
Closes #11787

### CODE REVIEW

**Comment by Erisa** [comment]
For some reason the preview doesn't work: https://erisa-cache-api-limits.cloudflare-docs-7ou.pages.dev/workers/platform/limits/

But it does when I `npm run dev` locally: 
image

**Comment by Erisa** [comment]
After the latest commit it shows in the preview URL now

**Review by irvinebroque — APPROVED** [review]

File: content/workers/platform/limits.md [hunk 1]
```diff
@@ -91,11 +91,11 @@ On the Unbound billing model, scheduled Workers ([Cron Triggers](/workers/config
...
-| Feature                       | Free   | Bundled |
-| ----------------------------- | ------ | ------- |
-| [Max object size](#cache-api-limits) | 512 MB | 512 MB  |
-| [Calls/request](#cache-api-limits)   | 50     | 50      |
-| [Storage/request](#cache-api-limits) | 5 GB   | 5 GB    |
+| Feature                       | Workers Free  | [Bundled](/workers/platform/pricing/#bundled-usage-model) | [Unbound](/workers/platform/pricing/#unbound-usage-model) | [Standard](/workers/platform/pricing/#standard-usage-model)  |
+| ----------------------------- | ------------- | ------- | ------- | ------- | 
+| [Max object size](#cache-api-limits) | 512 MB | 512 MB  | 512 MB  | 512 MB  |
+| [Calls/request](#cache-api-limits)   | 50     | 50      | 1,000   | 1,000   |
+| [Storage/request](#cache-api-limits) | 5 GB   | 5 GB    | 5GB     | 5GB     |
```
Comment (deadlypants1973) [inline content/workers/platform/limits.md:98]:
```suggestion
| [Storage/request](#cache-api-limits) | 5 GB   | 5 GB    | 5 GB    | 5  GB    |
```

Comment (deadlypants1973) [inline content/workers/platform/limits.md]:
```suggestion
| [Storage/request](#cache-api-limits) | 5 GB   | 5 GB    | 5 GB    | 5 GB    |
```

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**